hnn_core/tests/test_network.py

```diff
@@ -17,6 +17,19 @@
 params_fname = op.join(hnn_core_root, 'param', 'default.json')
 
 
+def test_network_io(tmpdir):
+    net_jones = jones_2009_model()
+    # Writing network
+    net_jones.write(tmpdir.join('net_jones.hdf5'))
+    # Reading network
+    net_jones_read = read_network(tmpdir.join('net_jones.hdf5'))
+    # Check cell_types
+    # assert net_jones.cell_types == net_jones_read.cell_types
+    print(net_jones.cell_types)
+    print(net_jones_read.cell_types)
```

Comment:
For test-driven development, you can use `pytest` with `--pdb` to directly drop into the interpreter when a test fails. Then you can use `pdb` commands like `p net_jones.cell_types` etc. Just letting you know the different options on debugging efficiently